src/Storage/Clickhouse.php

```diff
@@ -63,6 +63,11 @@ public function last($count, DiagRecord $beforeRecord = null): array
 
     public function batch(array $data): bool
     {
+        foreach ($data as $record) {
+            if (!($record instanceof Record)) {
+                throw new \RuntimeException('items passed to "batch" method should be instanceof ' . Record::class);
```

Comment:
decided to leave it here, because if in the $data array a non Record element passed, then it's invalid usage of the "batch" method within the diag application